.github/workflows/validate-swagger.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+      tags:
+        description: 'Validate Swagger.yaml'
+
+jobs:
+  validate_swagger:
+    runs-on: ubuntu-latest
+    name: Validate Swagger
+    steps:
+      - uses: actions/checkout@v2
+      - name: Validate OpenAPI definition
+        run: yarn validate
```

Comment:
nit: add a newline

Reply:
Fixed in [c481189](https://github.com/fiatconnect/specification/pull/41/commits/c4811894801a762d29f686e88dea4b2ece5a3e1e)!